src/rofl/common/cthread.cpp

```diff
@@ -298,107 +298,78 @@ cthread::clear_timers()
 {
 	AcquireReadWriteLock lock(tlock);
 	ordered_timers.clear();
-	for (auto it : timers)
-		delete it.second;
-	timers.clear();
 };
```

Comment:
Just to unify the code, can we get rid of the `;` after the `}` closing the function (and all other functions remaining)?